scmrepo/git/__init__.py

```diff
@@ -348,7 +348,7 @@ def add_commit(
     iter_refs = partialmethod(_backend_func, "iter_refs")
     iter_remote_refs = partialmethod(_backend_func, "iter_remote_refs")
     get_refs_containing = partialmethod(_backend_func, "get_refs_containing")
-    push_refspec = partialmethod(_backend_func, "push_refspec")
+    push_refspecs = partialmethod(_backend_func, "push_refspecs")
```

Comment:
@karajan1001 @pmrowla Is renaming intentional? I see that we use `push_refspec` in dvc, so this release seems to break experiments in dvc

Reply:
Ah, ok, just need to update dvc to work with 0.0.19 (we have it pinned in dvc to 0.0.18), no biggie. Sorry for the noise.

Reply:
Correction: I just didn't rebase in awhile and didn't notice that it was fixed already quite some time ago. Thank you :)